dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -415,6 +415,116 @@ export const Explorer = ({
     }
   };
 
+  const fetchDataUpdateChart = async (
+    startingTime?: string,
+    endingTime?: string,
+    updatedQuery?: any
+  ) => {
+    console.log('IN FETCH DATA TEST@@@@ ===EXPLORER ======== updatedQuery', updatedQuery);
+    const curQuery = updatedQuery.query;
+    console.log('curQuery====', curQuery, 'appBasedRef.current', appBasedRef.current);
+    // const rawQueryStr = buildQuery(appBasedRef.current, curQuery![RAW_QUERY]);
```

Comment:
you may remove loggings/comments next time before committing or merge these in one commit that only contains logging/comments removed version so it could help to remove noice in PR review